Update to core v0.11 (#29)
membraneframework/membrane_audio_mix_plugin: README.md

Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.

```elixir
  {:membrane_audio_mix_plugin, "~> 0.12.0"}
```

## Description

Both elements operate only on raw audio (PCM), so some parser may be needed to precede them in a pipeline.

> Audio format can be set as an element option or received through stream format from input pads. All
> stream format received from input pads have to be identical and match ones in element option (if that
> option is different from `nil`).

Reviewer: A simple regex isn't enough as it doesn't sound right now. BTW I realized this is a bit outdated since 0.10 as each element must send ~caps~ stream format struct, so I'd write something like

```suggestion
The elements expect to receive the same audio stream format on each input pad. It can be additionally enforced by setting an element option (`:stream_format`)
```

Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.

```elixir

> 	{:membrane_audio_mix_plugin, "~> 0.12.0"}

Reviewer: tab -> spaces
```suggestion
  {:membrane_audio_mix_plugin, "~> 0.12.0"}
```

  alias Membrane.File.{Sink, Source}

  @impl true
  def handle_init(_ctx, {path_to_wav_1, path_to_wav_2}) do
    structure = [
      child({:file, 1}, %Source{location: path_to_wav_1})
      |> child({:parser, 1}, Membrane.WAV.Parser)
      |> get_child(:interleaver),

      child({:file, 2}, %Source{location: path_to_wav_2})
      |> child({:parser, 2}, Membrane.WAV.Parser)
      |> get_child(:interleaver),

      child(:interleaver, %Membrane.AudioInterleaver{
        input_stream_format: %Membrane.RawAudio{
          channels: 1,
          sample_rate: 16_000,
          sample_format: :s16le
        },
        order: [:left, :right]
      })
      |> child(:file_sink, %Sink{location: "output.raw"})
    ]

    {[spec: structure], %{}}
  end
end

```

## Copyright and License

> Copyright 2023, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane)

Reviewer: You shouldn't change that - it indicates the year copyrights were claimed, i.e. year of the first publish
```suggestion
Copyright 2021, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane)
```
